test/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -38,37 +40,98 @@ class CheckYourAnswersControllerSpec extends SpecBase with SummaryListFluency wi
 
     "for a GET" - {
 
-      "must return OK and the correct view" in {
+      "must return OK and the correct view with valid non-optional data" in {
```

Comment:
It took me a bit to understand this description and the one below.
Maybe update to something more specific like with mandatory fields present
and the one below to with optional fields present? Not sure exactly how to best word it but it's a little confusing to me right now :D